test/pummel/test-hash-seed.js

```diff
@@ -1,31 +1,25 @@
 'use strict';
 
 // Check that spawn child doesn't create duplicated entries
-require('../common');
-const Countdown = require('../common/countdown');
+const common = require('../common');
 const REPETITIONS = 2;
 const assert = require('assert');
 const fixtures = require('../common/fixtures');
-const { spawn } = require('child_process');
+const { promisify } = require('util');
+const { execFile } = require('child_process');
+const execFilePromise = promisify(execFile);
 const targetScript = fixtures.path('guess-hash-seed.js');
-const seeds = [];
 
-const requiredCallback = () => {
+const requiredCallback = common.mustCall((results) => {
+  const seeds = results.map((val) => val.stdout.trim());
   console.log(`Seeds: ${seeds}`);
   assert.strictEqual(new Set(seeds).size, seeds.length);
   assert.strictEqual(seeds.length, REPETITIONS);
-};
+});
 
-const countdown = new Countdown(REPETITIONS, requiredCallback);
+const generateSeed = () => execFilePromise(process.execPath, [targetScript]);
+const subprocesses = [...new Array(REPETITIONS)].map(generateSeed);
```

Comment:
```suggestion
const subprocesses = (new Array(REPETITIONS)).map(generateSeed);
```

Not certain why the `[... ]` wrapping is necessary here?

Reply:
```console
> new Array(5).map(() => 'foo')
[ <5 empty items> ]
> [...new Array(5)].map(() => 'foo')
[ 'foo', 'foo', 'foo', 'foo', 'foo' ]
> 
```

Reply:
(It's because `[...new Array(5)]` results in an array with five items in it, all set to `undefined` whereas `new Array(5)` results in an array with 5 empty elements, which isn't the same thing and won't be iterated over by `map()`.

Reply:
ah, that's right. Forgot about that... Carry on, don't mind me.